src/backend/hyperplay_store_preload.ts

```diff
@@ -18,6 +19,7 @@ contextBridge.exposeInMainWorld('api', {
   apiVersion: 1,
   appIsInLibrary: async (gameId: string) =>
     ipcRenderer.invoke('appIsInLibrary', gameId, 'hyperplay'),
-  goToGamePage: (gameId: string) => ipcRenderer.send('goToGamePage', gameId),
+  goToGamePage: (args: { gameId: string; action: GamePageActions }) =>
```

Comment:
```suggestion
  goToGamePage: (gameId: string; action?: GamePageActions ) =>
```

if we define it like this, we don't need to worry about backwards compatibility of the store with old versions of the client

Reply:
What if I told you that I wrote like this at first and then I thought that on the code reviews you guys would ask me to change to use the object instead? 🥲🤣🤣🤣🤣

Ok, will update those.

Reply:
done